api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -279,6 +279,15 @@ message ExtAuthz {
   // correctness checks for all header / query parameter mutations (e.g. for invalid characters).
   // This field allows the filter to reject mutations to specific headers.
   config.common.mutation_rules.v3.HeaderMutationRules decoder_header_mutation_rules = 26;
+
+  // Disables ingestion of dynamic metadata from ext_authz service.
+  //
+  // If set to true, the filter will ignore the dynamic_metadata field in the ext_authz response. If
+  // it is non-empty, the filter will log, increment the ``ignored_dynamic_metadata`` stat, then
+  // continue handling the response.
+  //
+  // If set to false (the default value), the filter will ingest dynamic metadata entries as normal.
+  bool disable_dynamic_metadata_ingestion = 27;
```

Comment:
It's always a little weird when there's a bool where a true value means to disable something instead of enabling it.  Instead, how about this:

```
// Controls whether the filter ingests the dynamic_metadata field in the ext_authz response.
// If unset, defaults to true.
google.protobuf.BoolValue enable_dynamic_metadata_ingestion = 27;
```

Reply:
Agreed, done.